### DISCUSSION TITLE
create admin:org scope aceess token only for specific repo in github

### DISCUSSION THREAD

**Discussion by ghost** [community/community#29907] [Other Feature Feedback, Questions, & Ideas]
Hello team

We are using GitHub Actions.
To access the self-hosted runners API needed admin:org scope aceess token
For GitHub actions I am creating a personal access token

Access tokens require admin:org scope
```

 admin:org                     Full control of orgs and teams, read and write org projects
 write:org                       Read and write org and team membership, read and write org projects
 read:org                       Read org and team membership, read org projects
 manage_runners:org   Manage org runners and runner groups

```
With this token we can access all git repos that is avalable in my organazaion. To avaoid misused please help me to create admin token for specific repo with that i can able to use github runners api and all features of github actions.

![Screenshot from 2022-08-18 13-04-15](https://user-images.githubusercontent.com/87641643/185340608-91876916-e4f6-4c4e-9b49-876872ee5559.png)

**Comment by bogedal** [comment]
Can you create this admin token and place it in the organization secret and limit this secret to be accessed only from the repo?

That is how I have setup the org:admin token.
